lib/mrkt/concerns/crud_custom_activities.rb

```diff
@@ -0,0 +1,39 @@
+require 'time'
+
+module Mrkt
+  module CrudCustomActivities
+    def get_list_of_custom_activity_types()
```

Comment:
Please use the coding style of the project, in this case do not add parens if there are no arguments for a method.